examples/examples_test.go

```diff
@@ -62,8 +62,10 @@ func TestServiceConnectionIdDefaultDoesNotConflict(t *testing.T) {
 			"response": {
 				"inputs": {
 					"__defaults": [
+						"disabled",
 						"name"
 					],
+					"disabled": false,
```

Comment:
@iwahbe does this look okay to you - had to update this replay test ... looks like just a new default value coming in so should be fine right?

Reply:
Looks fine to me.